ql/experimental/averageois/averageoiscouponpricer.hpp

```diff
@@ -37,9 +37,9 @@ namespace QuantLib {
                                          : public FloatingRateCouponPricer {
     public:
         ArithmeticAveragedOvernightIndexedCouponPricer(
+            bool byApprox = false, // TRUE to use Katsumi Takada approximation
             Real meanReversion = 0.03,
-            Real volatility = 0.00, // NO convexity adjustment by default
-            bool byApprox = false) // TRUE to use Katsumi Takada approximation
+            Real volatility = 0.00) // NO convexity adjustment by default
```

Comment:
This is in `experimental` so it's not a big problem to change an interface, but is there a reason why you reordered the parameters?  Do you think `byApprox` is more likely to be passed than `meanReversion`?

Reply:
I reordered it, because in the `overnightindexedcoupon.cpp` I could conveniently pass the `telescopicValueDates` to the constructor of `ArithmeticAveragedOvernightIndexedCouponPricer`. Now that you pointed this out, I believe I could have maybe better overloaded the constructor, instead of changing the interface. Sorry about that.